hightouch_blueprints/execute_sync.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+import errors
```

Comment:
will do

Reply:
So unfortunately, because of the way we pull the code into Shipyard, it will have to be imported this way.

# Handle import difference between local and github install
try:
    import errors
except BaseException:
    from . import errors